sites/all/modules/custom/practice/css/practice.css

```diff
@@ -0,0 +1,6 @@
+tr.even{
```

Comment:
add a space between CSS selector and the opening brace
tr.even {

Reply:
https://www.drupal.org/node/1887862#whitespace - The opening brace should include a single space before it.